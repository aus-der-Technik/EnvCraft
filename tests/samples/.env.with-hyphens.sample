# Test variable name with hyphens
api-key=
