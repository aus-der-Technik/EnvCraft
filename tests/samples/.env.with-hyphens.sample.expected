api-key=my-shiny-new-api-key
